zanata-war/src/main/java/org/zanata/events/ProjectUpdate.java

```diff
@@ -35,4 +35,6 @@
     public static final String EVENT_NAME = "org.zanata.events.ProjectUpdate";
 
     HProject project;
+
+    String oldSlug;
```

Comment:
Yes see above. I needed the old slug so that I can look up existing workspace for editor (the key in hashmap uses old slug) and send events to them.